app_legacy/Helpers/database/player-game.php

```diff
@@ -436,13 +436,19 @@ function getTotalUniquePlayers(int $gameID, ?string $requestedBy = null, bool $h
         $requestedByStatement = 'OR ua.User = :requestedBy';
     }
 
+    if ($parentGameID !== null) {
+        $bindings['parentGameId'] = $parentGameID;
+    }
+
+    $gameIdStatement = $parentGameID === null ? 'ach.GameID = :gameId' : 'ach.GameID IN (:gameId, :parentGameId)';
```

Comment:
suggest merging this into the above if:
```
$gameIdStatement = 'ach.GameID = :gameId';
if ($parentGameID !== null) {
    $gameIdStatement = 'ach.GameID IN (:gameId, :parentGameId)';
    $bindings['parentGameId'] = $parentGameID;
}

Reply:
Done in https://github.com/RetroAchievements/RAWeb/pull/1530/commits/c384be10eec370633b6d37d8f2c33714547b872f